timemachine/md/builders.py

```diff
@@ -19,7 +21,8 @@ def build_protein_system(host_pdbfile):
         PDB of the host structure
 
     """
-    host_ff = app.ForceField("amber99sbildn.xml", "tip3p.xml")
+
+    host_ff = app.ForceField(f"{tm_ff.protein_ff}.xml", f"{tm_ff.water_model}.xml")
```

Comment:
Should the users have to add the extension?

Reply:
The extension seems like an implementation detail. We may switch formats if we start fitting the water/protein models.